pkg/users/account-create-dialog.js

```diff
@@ -240,6 +277,13 @@
                         prog.push('-c');
                         prog.push(real_name);
                     }
+
+                    prog.push('-u');
+                    prog.push(uid);
+
+                    if (force_uid)
+                        prog.push('-o'); // Create user with non-unique user-specified UID, useful at certain use cases
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18353-20230215-154634-5d29fa30-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)